OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -363,4 +355,11 @@
     <Message Text="DEBUG TargetName:             $(TargetName)"/>
     -->
   </Target>
-</Project>
+  <PropertyGroup>
+    <PostBuildEvent>IF  EXIST  "%25VS140COMNTOOLS%25"  CALL  "%25VS140COMNTOOLS%25vsvars32.bat"
```

Comment:
Yes, do not use `PostBuildEvent` with embedded batch.

We, of course, do not want this to run on non-Windows OSes.

Instead use MSBuild macros / tasks, hopefully these links help:
https://msdn.microsoft.com/en-us/library/dd293663.aspx
http://stackoverflow.com/questions/802070/how-do-i-check-if-any-file-in-an-item-list-exist-using-msbuild
https://msdn.microsoft.com/en-us/library/x8zx72cd.aspx